src/wipplpy/modules/shot_loader.py

```diff
@@ -0,0 +1,179 @@
+"""Load a shot from an MDSplus tree using a local or remote connection."""
+import json
+import logging
+import os
+
+import MDSplus as mds
+from MDSplus.mdsExceptions import MDSplusException, SsSUCCESS
+
+# TODO: Add MySQL Connection object.
+_mds_connection = None
+_global_tree = None
+
+
+def get_connector(server_name, reconnect=False):
+    """
+    Get the MDSplus connector for a remote connection.
+
+    Parameters
+    ----------
+    server_name : str
+        Server ip address.
+    reconnect : bool, default=False
+        Whether to force a reconnection to the server.
+
+    Returns
+    -------
+    mds.Connection
+        Connection to the server without a tree connection.
+    """
+    global _mds_connection
```

Comment:
```suggestion
    global _mds_connection  # noqa: PLW0603
```
The `# noqa` comment stands for "no quality assurance" and the `PLW0604` is the specific [ruff rule](https://docs.astral.sh/ruff/rules/#pylint-pl) to ignore for this specific line.